README.md

```diff
@@ -77,6 +77,24 @@ bin/elixirc lib/elixir/lib/string.ex -o lib/elixir/ebin
 bin/elixir lib/elixir/test/elixir/string_test.exs
 ```
 
+To recompile only changed files (including Erlang modules):
+
+```sh
+make compile
+```
+
+If you find yourself seeing errors during the bootstrap compilation step, you may need to re-boostrap Elixir:
```

Comment:
"If your contribution fails the build during the bootstrapping of the language, you can reproduce it locally by deleting all of Elixir beam files and compiling again:"